src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div
+        class="card container__card"
+        data-qa="card"
+      >
+        <p class="title card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
```

Comment:
```suggestion
        <p class="title card__title">
          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
        </p>
```
Добавляй отступ
Этот текст можно сделать заголовком(h2 например)